src/main.c

```diff
@@ -430,6 +439,14 @@ void create_command_list(CommandList *c) {
                                     "password=-",
                                     fmt_password, fmt_nofmt,
                                     "Store an authentication key", NULL, NULL});
+  register_subcommand(*c,
+                      (Command){"authkey_asym", yh_com_put_authentication_asym,
```

Comment:
I know this would be somewhat "backward incompatible" but I think that having `put authkey` take an algorithm parameter would be better than having `put authkey_asym`.

I still haven't made up my mind if this makes sense for other commands like `session open` though.

Reply:
Just felt that this way had less impact on existing code, as well as on backwards compatibility usage wise.